hw4/README.txt

```diff
@@ -0,0 +1,6 @@
+1) Install Python 3.8 (https://www.python.org/downloads/)
+2) Install pytest library (pip install pytest)
+3) Open command line or PyCharm terminal (optional)
+4) Navigate ROOT directory of a project (use 'cd' command to navigate in Pycharm terminal)
+5) Run 'pytest' command
```

Comment:
By default pytest does not run doctests. Why does it work?

Reply:
Pytest is running test files in the current directory and its subdirectories

Reply:
My pytest.ini file have this line, that activates it
addopts = --doctest-modules --doctest-continue-on-failure